src/components/calcite-dropdown/calcite-dropdown.scss

```diff
@@ -1,77 +1,59 @@
 :host {
-  position: relative;
-  display: inline-flex;
-  flex-grow: 1;
+  @apply relative flex-grow;
 }
 
 // disabled styles
 :host([disabled]) {
-  pointer-events: none;
-  opacity: var(--calcite-ui-opacity-disabled);
+  @apply pointer-events-none opacity-disabled;
 }
-
 :host .calcite-dropdown-wrapper {
   @include popperContainer();
   @include popperWrapper();
 }
-
 @include popperElemAnim(".calcite-dropdown-wrapper");
-
 :host([active]) .calcite-dropdown-wrapper {
   @include popperWrapperActive();
 }
-
 :host .calcite-dropdown-content {
-  background: var(--calcite-ui-foreground-1);
+  @apply bg-foreground-1
+    overflow-hidden
+    overflow-x-auto
```

Comment:
`overflow-y-auto` (unless `overflow-x-auto` is intentional?)

Reply:
good catch!